Src/Support/Google.Apis.Auth.Tests/GoogleJsonWebSignatureTests.cs

```diff
@@ -165,7 +161,7 @@ public async Task Validate_Signature_Time()
         [Fact]
         public async Task Invalid_HostedDomain()
         {
-            var clock = new MockClock() { UtcNow = new DateTime(2017, 5, 31, 10, 24, 0, DateTimeKind.Utc) };
+            var clock = new MockClock() { UtcNow = FakeCertificateCache.ValidJwtGoogleSigned };
```

Comment:
Aside: we should really add a `MockClock(DateTime)` constructor. Let's do that in a separate PR.